api/policies/accessTokenAuth.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function(req, res, next) {
+  AccessTokenAuth.checkAndSetAuthenticated(req).then(() => next())
```

Comment:
add a `.catch(err => ...)` with `res.serverError` or something like that, otherwise this could hang the response